vortex/core/helpers.py

```diff
@@ -27,3 +27,37 @@ def input_size(job):
 def weighted_random_sampling(destinations):
     rankings = [(d.params.get('weight', 1) if d.params else 1) for d in destinations]
     return random.choices(destinations, weights=rankings, k=len(destinations))
+
+
+def get_keys_from_dict(dl, keys_list):
```

Comment:
We probably don't want to expose this method through helpers?
```suggestion
def __get_keys_from_dict(dl, keys_list):
```